packages/nbd/nbd.2.1.3/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "1.2"
+maintainer: "dave@recoil.org"
+authors: ["Jonathan Ludlam" "David Scott"]
+license: "LGPL-2 with OCaml linking exception"
+homepage: "https://github.com/xapi-project/nbd"
+dev-repo: "https://github.com/xapi-project/nbd.git"
+bug-reports: "https://github.com/xapi-project/nbd/issues"
+build: [
+  ["oasis" "setup"]
+  ["./configure" "--prefix" prefix]
+  [make]
+]
+build-test: [
+  ["./configure" "--enable-tests"]
+  [make "test"]
+]
+install: [
+  [make "install" "BINDIR=%{bin}%"]
+]
+remove: [
+  [make "uninstall" "BINDIR=%{bin}%" "CONFIGUREFLAGS=--disable-tests"]
+  ["ocamlfind" "remove" "nbd"]
+]
+depends: [
+  "oasis" {build}
+  "ounit" {test}
+  "ocamlfind" {build}
+  "ppx_tools" {build}
+  "lwt" {>= "2.7.0" & < "3.0.0"}
+  "cstruct" {>= "1.9.0"}
+  "cmdliner"
+  "sexplib"
+  "mirage-block-unix" {< "2.5.0"}
+  "io-page"
+  "mirage" {>= "1.1.0" & < "3.0.0"}
```

Comment:
I think this needs to be `mirage-types` and not `mirage`, otherwise this error is happening:

```
#=== ERROR while installing nbd.2.1.3 =========================================#
# opam-version 1.2.2
# os           linux
# command      make
# path         /home/travis/.opam/4.04.0/build/nbd.2.1.3
# compiler     4.04.0
# exit-code    2
# env-file     /home/travis/.opam/4.04.0/build/nbd.2.1.3/nbd-25813-20b76c.env
# stdout-file  /home/travis/.opam/4.04.0/build/nbd.2.1.3/nbd-25813-20b76c.out
# stderr-file  /home/travis/.opam/4.04.0/build/nbd.2.1.3/nbd-25813-20b76c.err
### stdout ###
# [...]
# /home/travis/.opam/4.04.0/bin/ocamlfind ocamldep -package cstruct -package cstruct.ppx -package io-page -package lwt -package mirage-types.lwt -package ppx_sexp_conv -package sexplib -modules lib/mirror.ml > lib/mirror.ml.depends
# /home/travis/.opam/4.04.0/bin/ocamlfind ocamlc -c -g -annot -bin-annot -package cstruct -package cstruct.ppx -package io-page -package lwt -package mirage-types.lwt -package ppx_sexp_conv -package sexplib -I lib -o lib/protocol.cmo lib/protocol.ml
# /home/travis/.opam/4.04.0/bin/ocamlfind ocamlc -c -g -annot -bin-annot -package cstruct -package cstruct.ppx -package io-page -package lwt -package mirage-types.lwt -package ppx_sexp_conv -package sexplib -I lib -o lib/client.cmo lib/client.ml
# + /home/travis/.opam/4.04.0/bin/ocamlfind ocamlc -c -g -annot -bin-annot -package cstruct -package cstruct.ppx -package io-page -package lwt -package mirage-types.lwt -package ppx_sexp_conv -package sexplib -I lib -o lib/client.cmo lib/client.ml
# File "lib/client.ml", line 1:
# Error: The implementation lib/client.ml
#        does not match the interface lib/client.cmi:
#        The value `connect' is required but not provided
#        The value `id' is required but not provided
# Command exited with code 2.
### stderr ###
# W: Cannot find source file matching module 'nbd' in library nbd.
# W: Use InterfacePatterns or ImplementationPatterns to define this file with feature "source_patterns".
# E: Failure("Command ''/home/travis/.opam/4.04.0/bin/ocamlbuild' lib/nbd.cma lib/nbd.cmxa lib/nbd.a lib/nbd.cmxs lwt/nbd-lwt.cma lwt/nbd-lwt.cmxa lwt/nbd-lwt.a lwt/nbd-lwt.cmxs cli/main.native -tag debug' terminated with error code 10")
# make: *** [build] Error 1
```